swagger-spec/preprints/institutions_list.yaml

```diff
@@ -0,0 +1,102 @@
+#/preprints/{preprint_id}/institutions/
+get:
+  summary: List all Institutional Affilations for a Preprint
+  description: >-
+    A paginated list of the Preprint's Institutions, sorted by their index.
+
+    Institutions are organizations affiliated with the Preprint. Contributors may be associated with one or more institutions.
+
+    #### Returns
+
+    Returns a JSON object containing `data` and `links` keys.
+
+    The `data` key contains an array of institutions.
+    Each resource in the array contains the full representation of the institution, meaning additional requests to an institution's detail view are not necessary.
+    Additionally, the full representation of the institution is automatically embedded within the `data` key of the response.
+
+    The `links` key contains a dictionary of links that can be used for [pagination](#tag/Pagination).
+
+    #### Filtering
+
+    You can optionally request that the response only include institutions that match your filters by utilizing the
+    `filter` query parameter, e.g. https://api.osf.io/v2/preprints/y9jdt/institutions/?filter[name]=example.
+
+  parameters:
+  - in: path
+    type: string
+    name: preprint_id
+    required: true
+    description: 'The unique identifier of the preprint.'
+  tags:
+  - Preprints
+  operationId: preprints_institutions_list
+  x-response-schema: Institution
+  responses:
+    '200':
+      description: OK
+      schema:
+        type: array
+        items:
+          $ref: 'definition.yaml'
+      examples:
+        application/json:
+          data:
+            - relationships:
+                nodes:
+                  links:
+                    related:
+                      href: https://api.osf.io/v2/institutions/cos/nodes/
+                      meta: {}
+                users:
+                  links:
+                    related:
+                      href: https://api.osf.io/v2/institutions/cos/users/
+                      meta: {}
+                registrations:
+                  links:
+                    related:
+                      href: https://api.osf.io/v2/institutions/cos/registrations/
+                      meta: {}
+                department_metrics:
+                  links:
+                    related:
+                      href: https://api.osf.io/v2/institutions/cos/metrics/departments/
+                      meta: {}
+                users_metrics:
+                  links:
+                    related:
+                      href: https://api.osf.io/v2/institutions/cos/metrics/users/
+                      meta: {}
+                summary_metrics:
+                  links:
+                    related:
+                      href: https://api.osf.io/v2/institutions/cos/metrics/sumary/
+                      meta: {}
+                    data:
+                      id: cos
+                      type: institution-summary-metrics
```

Comment:
ditto for possibly removing the metrics endpoints whose contents are only visible to institution admins